cdap-ui/app/features/flows/templates/detail.html

```diff
@@ -37,16 +37,16 @@ <h2 class="programTitle">
         <span class="caret"></span>
       </button>
 
-      <ul class="dropdown-menu" role="menu" ng-if="status === 'STOPPED'">
+      <ul class="dropdown-menu dropdown-menu-right" role="menu" ng-if="status === 'STOPPED'">
         <li>
           <a href="#" ng-click="openRuntime()">
-            <span class="fa fa-wrench"></span>
+            <span class="fa fa-wrench pull-right"></span>
             Runtime Arguments
```

Comment:
If we are adding the icon in span, I feel we should wrap the text in span too. (Makes it slightly easier to style in the future) Or we could use ```<i>``` tags for the icons.